presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java

```diff
@@ -684,7 +688,14 @@ public boolean last()
     public int getRow()
             throws SQLException
     {
-        throw new SQLFeatureNotSupportedException("getRow");
+        checkOpen();
+
+        long rowNumber = currentRowNumber.get();
+        if (rowNumber < 0 || rowNumber > Integer.MAX_VALUE) {
+            throw new SQLException("Max rows exceeds limit of 2147483647");
```

Comment:
"Max rows" looks incorrect. Could you change to "Rows exceed ..." or "Current row exceed ..."?

Reply:
Oh, yeah. Sorry for this.